src/authenticationinapp/authenticationinapplistener.cpp

```diff
@@ -754,6 +756,9 @@ void AuthenticationInAppListener::processErrorCode(int errorCode) {
     case 998:  // An internal validation check failed.
       [[fallthrough]];
     default:
+      emit MozillaVPN::instance()->recordGleanEventWithStringType(
+          GleanSample::authenticationInappError, "errno",
+          QString::number(errorCode));
```

Comment:
You've gone to all the trouble of typing out the error names. I'm wondering if it'll be easier for future analysis if you pass a string that concatenates the error code and description?

Reply:
What about if we do not have them all?
The errno should be enough because we can fetch the description for the API documentation https://github.com/mozilla/fxa/blob/main/packages/fxa-auth-server/docs/api.md  - if a new error code is added, our code does not break.